src/coreclr/inc/corinfo.h

```diff
@@ -954,11 +954,14 @@ enum CorInfoClassId
 
 enum CorInfoInline
 {
-    INLINE_PASS                 = 0,    // Inlining OK
+    INLINE_PASS                     = 0,    // Inlining OK
+    INLINE_PREJIT_SUCCESS           = 1,
+    INLINE_CHECK_CAN_INLINE_SUCCESS = 2,
+    INLINE_CHECK_CAN_INLINE_VMFAIL  = 3,
```

Comment:
IMO, it would be nice to have some comments for each of these values. E.g., I don't know how to read `INLINE_CHECK_CAN_INLINE_SUCCESS`.

Reply:
Added.